src/main/java/dev/slimevr/vr/trackers/TrackersUDPServer.java

```diff
@@ -234,7 +231,7 @@ public void run() {
 					case 17: // PACKET_ROTATION_DATA
 						if(connection == null)
 							break;
-						if(connection.isOwoTrack)
+						if(connection.firmware.equals("owoTrack"))
```

Comment:
Please revert it to boolean check, this boolean should be set on tracker connect, and not string check at every packet.

Reply:
I did this because this check doesn't apply to owoTrack for Android as it reports `owoTrack8` in firmware (haven't checked for iOS), and compatibility with both versions cannot be expressed with a single boolean check. If you're concerned by string comparison performance, then see updated version with enum.